filters/badware.txt

```diff
@@ -3451,7 +3451,7 @@ torrdroidforpc.com##[href^="http://slugmefilehos.xyz/"]
 ||fitgirlrepacksite.com^$all,domain=~fitgirl-repacks.site
 ! https://github.com/uBlockOrigin/uAssets/issues/18206
 ||fitgirl.cc^$all,domain=~fitgirl-repacks.site
-! [replace with link to PR]
+! https://github.com/uBlockOrigin/uAssets/issues/21658
```

Comment:
That can by calculated based on last issue/pr/disscussion based on frequency of addition these 3 at all at day. 

Also you generated 302:

```yml
>>> https://github.com/uBlockOrigin/uAssets/issues/21658

> --------------------------------------------
> 302 Found
> --------------------------------------------
Status:	302 Found
Code:	302
Server:	GitHub.com
Date:	Fri, 29 Dec 2023 22:06:59 GMT
Content-Length:	0
Location:	https://github.com/uBlockOrigin/uAssets/pull/21658
server-timing:	nginx;desc="NGINX";dur=1.207795,glb;desc="GLB";dur=44.914835
x-voltron-version:	6f4c7bf
Vary:	Accept-Encoding, Accept, X-Requested-With
X-Frame-Options:	DENY
Set-Cookie:	logged_in=no; Path=/; Domain=github.com; Expires=Sun, 29 Dec 2024 22:06:59 GMT; HttpOnly; Secure; SameSite=Lax
X-GitHub-Request-Id:	B296:368625:CEFEFBD:D1E5171:658F4303
connection:	close
```

Reply:
That's what I ended up doing, I just though of that after I submitted the original commit.
> Also you generated 302:

Good catch. However, I can't fix it and IMO it's not important enough to trouble someone with commit access.
Thanks

Reply:
https://github.com/uBlockOrigin/uAssets/pull/21658#issuecomment-1872360935

Reply:
> What is good to allow scam images at official page

If I understand you correctly, part of the idea of adding the `$domain=~fitgirl-repacks.site` is to show users what the correct site is (kind of like Imre's hacky use of $domain for block messages):
![image](https://github.com/uBlockOrigin/uAssets/assets/84232764/78a990c4-0061-4738-94f3-c43fbc0d669e)
Thanks